### ISSUE TITLE
Generating CHANGELOG.md

### ISSUE THREAD

**Issue by gunnx** [ngryman/cz-emoji#1]
Have you used this to generate a changelog file? 
Semantic release and standard version expect the angular preset but this adapter uses the emoji value

**Comment by ngryman** [comment]
Hey @gunnx,

If you generate a `changelog.md` file from your `git` history, then you'll have types of commit represented as emojis instead of text. That's the whole point of this adapter.

I'm not sure I understood your question correctly tho :sweat_smile:

**Comment by gunnx** [comment]
Hi,
I've been recently looking into standardised commits and auto generation of changelog, I was looking at 'standard-version' or conventional changelog and they follow the angular style so expect keywords like 'feat', 'bug' etc 
I just wondered if you knew of any tools that can be customised so you can generate a changelog using this emoji adapter.

Thanks

**Comment by ngryman** [comment]
Oh ok, you are speaking about [conventional changelog]. I didn't understand correctly, sorry.

From what I understand, [conventional changelog] can be customized the way you want: https://github.com/conventional-changelog/conventional-changelog#options. You could configure it to process emojis instead.

Keep me posted if you do so, so I could mention it in the `readme` for other developers with your needs. Thanks!

I'm closing this issue as it's out of this project scope.

[conventional changelog]: https://github.com/conventional-changelog/conventional-changelog

**Comment by tpucci** [comment]
Hi, I use cz-emoji on every of my projects.
I created a npm package to generate a conventional changelog when one uses cz-emoji :)
https://www.npmjs.com/package/conventional-changelog-cz-emoji
I'm interested by feedbacks.

**Comment by crhistianramirez** [comment]
If anyone else is stumbling upon this, I'm using [semantic-release](https://github.com/semantic-release/semantic-release) which does some cool stuff with the standardized commit templates, one of those is generate a changelog. The default commit template it understands is of course the angular one however there is [this plugin](https://github.com/momocow/semantic-release-gitmoji) specifically for parsing the gitmoji standard.  

So to summarize I use cz-emoji to create standardized commits, and then semantic-release with the help of semantic-release-gitmoji can then do stuff to that such as generate changelogs or bump the semantic version of the package for me.
